### DISCUSSION TITLE
Why does N+ create gdbus-nonce-file-****** files in my \Local\Temp folder?

### DISCUSSION THREAD

**Discussion by DeathStalker77** [nicotine-plus/nicotine-plus#2703] [Q&A]
The asterisks indicate a random 5 alpha-numeric value.  What are they for and what do they do?

**Comment by mathiascode** [comment]
I believe they are used by gdbus somehow to identify and bring a running instance of Nicotine+ to the foreground when attempting to launch new ones. It's a shame the files don't seem to be removed though, I'll have to see if anything can be done regarding that.
